### PR TITLE
Allow user to copy TOTP URL during MFA #919

### PR DESCRIPTION
**Pull request by lennartkoopmann**
Some devices can't make use of the QR code (mobile devices for example). This change allows the user to manually copy the TOTP URL into their TOTP application.

### CODE REVIEW

**Comment by lennartkoopmann** [comment]
I forgot to set up commit signing on my home workstation. Will bypass branch protection for this one if required.

File: web-interface/src/components/system/authentication/MFASetupStep1.jsx [hunk 1]
```diff
@@ -21,9 +21,35 @@ function MFASetupStep1(props) {
...
+             tabIndex="-1"
+             aria-labelledby="otp-url-modal-label"
+             aria-hidden="true">
+          <div className="modal-dialog">
+            <div className="modal-content">
+              <div className="modal-body">
+                <h2>TOTP URL</h2>
+
+                <p>
+                  Many TOTP applications allow you to enter the TOTP URL directly f you cannot scan the QR code:
```
Comment (uskr) [inline web-interface/src/components/system/authentication/MFASetupStep1.jsx:39]: Typo on "f you cannot".
```suggestion
                  Many TOTP applications allow you to enter the TOTP URL directly if you cannot scan the QR code:
```

**Review by uskr — APPROVED** [review]
Looks good.

**Comment by lennartkoopmann** [comment]
Thank you!

**Pull request merged**